src/app/dashboard/dashboard.component.ts

```diff
@@ -49,4 +50,13 @@ export class DashboardComponent implements OnInit {
       return response.rows.map((item) => ({ ...item.doc, title: item.doc[titleField], link: linkPrefix + (addId ? item.id : '') }));
     }));
   }
+
+  filterResigned(res) {

```

Comment:
@paulbert  I think we should go with mango query to get the list of courses or library. And not try to filter it on user side. It will be fetching lot of data when we will only be showing few.